api/app/api.py

```diff
@@ -20,6 +30,65 @@
 )
 
 @app.post("/forecast/")
-async def forecast(site:PVSite):
-    df =run_forecast(site)
-    return df.to_dict()
\ No newline at end of file
+def forecast(request: ForecastRequest):
```

Comment:
could you rename request to forecast_request, request sometimes has a different meaning in apis (it can be the http request)

Reply:
> could you rename request to forecast_request, request sometimes has a different meaning in apis (it can be the http request)

Sure can do that

Reply:
Thank you